### DISCUSSION TITLE
Image view - Menu "Edit" opens blank editor window

### DISCUSSION THREAD

**Discussion by francwalter** [hluk/CopyQ#2362] [Q&A]
Hallo

I would like to open a image-clip in a bigger view somehow.
At the moment I have only the "Edit with Editor" menu entry where I can view the image in my Image software (on Windows I use irfanview). 
When I just click "edit" an empty editor dialog opens, but there I cannot edit an image. For text this editor is good (e.g. with FakeVim), but not for images.
"Show content" doesnt help neither.

Is there a way to just view the image in a bigger size when I "edit" the image?
Is there maybe a command for it?

Thanks for hints.
frank
CopyQ 7.0 on Windows 10 (64) or Ubuntu 22.04

EDIT: this Discussions site is abandoned 😢 

I do it now like this: I ignore that context menu entry 'Edit' and I only use 'Edit with Editor' for which I setup the shortcut 'Space' and also 'Ctrl+E' in Settings > Shortcuts > Application. For Text I use [Ted Notepad](https://jsimlo.sk/notepad/download.php) (a very small and super fast editor with some features) and for Images as I wrote already, I use [IrvanView](https://www.irfanview.com/64bit.htm) (an image viewer/editor which is very small and also super fast).
For 'Edit' I setup the shortcut 'Ctrl+Shift+E' but I never use it (as Ted Notepad is far better than the implemented editor, even better than FakeVim).

**Comment by GFDGIT** [comment]
Hello,

You can preview an image by press F7.

Or, you can write a command to auto select a  correct program to open the image/text.

**Comment by hluk** [comment]
It should be possible to set an image editor in Preferences (Items config tab; Images entry):
![image](https://github.com/hluk/CopyQ/assets/38054/3374ddbc-e244-460c-94c0-e8a08c9f21ca)

**Reply by francwalter** [reply]
Yes, this is what I did. But the simple 'Edit' does nothing (see my first post), only when I 'Edit with Editor'.
So I switched the shortcuts (see my first post).

So what is ment with 'Edit' in the menu when I have an image? Is there an inline editor for _images_?

**Reply by hluk** [reply]
Ah, sorry I misunderstood.

> So what is ment with 'Edit' in the menu when I have an image?

For images you need to pick "Edit with Editor" (Ctrl+E).

> Is there an inline editor for images?

No, the inline editor can only modify text and simpler rich text.

I should change the behavior so that selecting Edit for images would run the external (image) editor if the inline editor does not make sense (there is usually no text beside the image stored in the item).

**Reply by hluk** [reply]
Fix in #2379.
